.gitignore

```diff
@@ -35,3 +35,5 @@ plugins
 package-lock.json
 
 .env
+
+.sentryclirc
```

Comment:
Created during testing, I would leave it in just in case so we don't leak CLI credentials